dotnet.cmd

```diff
@@ -2,33 +2,38 @@
 setlocal
 
 set "__ProjectDir=%~dp0"
-set "__RepoRootDir=%~dp0..\..\"
+set "__RepoRootDir=%__ProjectDir%..\..\"
 
 rem Remove after repo consolidation
-if not exist "%__RepoRootDir%.dotnet-runtime-placeholder" ( set "__RepoRootDir=!__ProjectDir!" )
+if not exist "%__RepoRootDir%\.dotnet-runtime-placeholder" ( set "__RepoRootDir=%__ProjectDir%" )
 
 :: Clear the 'Platform' env variable for this session, as it's a per-project setting within the build, and
 :: misleading value (such as 'MCD' in HP PCs) may lead to build breakage (issue: #69).
 set Platform=
-set __ProjectDir=
 
 :: Don't resolve runtime, shared framework, or SDK from other locations to ensure build determinism
 set DOTNET_MULTILEVEL_LOOKUP=0
 
 :: Disable first run since we do not need all ASP.NET packages restored.
 set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
-:: Install dotnet
-call %~dp0\init-dotnet.cmd
-if NOT [%ERRORLEVEL%]==[0] (
+echo Installing dotnet using Arcade...
```

Comment:
nit: Arcade outputs echos if it installs the SDK anyway in tools.sh/ps1. If it's already present I don't think there should be a console output.

Reply:
Hmm, if I delete it now, it will cancel the current testing and trigger a new 2-hour run. Would you mind taking the change as is and I'll delete this line in CoreCLR after you finish the port (i.e. it will be there for the "real" integration)?

Reply:
sure. was just a nit.